mw-webapp/package.json

```diff
@@ -24,7 +24,9 @@
   "scripts": {
     "start": "vite",
     "build": "tsc && vite build",
```

Comment:
why do we need run tsc here if it is in other script?

Reply:
this command don't allow to build project if in the project exists mistakes.

Reply:
Deleted